src/04_printing.py

```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %s," % x, "y is %s," % round(y, 2), "z is \"%s\"" % z)
```

Comment:
Here is a simpler way to format:
`print("x is %d, y is %.2f, z is \"%s\"" %(x, y, z))`

the `%.2f` rounds it to 2 decimal places, if you want to round it to 4 decimals, you can do `%.4f`